### ISSUE TITLE
innerDiffs is not iterable

### ISSUE THREAD

**Issue by philohelp** [aws-amplify/amplify-cli#5689]
```
✖ An error occurred when pushing the resources to the cloud

innerDiffs is not iterable
There was an error pushing the API resource
TypeError: innerDiffs is not iterable
```

**Describe the bug**
I got this message on the CLI when I try to push a change in my graphQL schema. I had made the last changes day before yesterday and all seemed to be good. 

EDIT: I also enabled Point in time recovery in Dynamodb in some of my tables. I didn't think it could have an effect on amplify. Could it be the cause of all this ?

**Amplify CLI Version**
4.27.1

**To Reproduce**
Just try to push changes in the graphQL schema with `amplify api push`

**Expected behavior**
Update the schema accordingly.

**Desktop (please complete the following information):**
 - OS:Mac
 - Node Version. v10.16.3

**Comment by philohelp** [comment]
I finally went with `amplify pull` and back to business. 

Still, I'd really like to know what this `innerDiffs` error stands for. 

Thanks in advance for your reply.

**Comment by edwardfoyle** [comment]
@philohelp Hard to say exactly what went wrong, but the innerDiff variable is referenced [here](https://github.com/aws-amplify/amplify-cli/blob/4faaba0509467aad03db11709175f4a3071459ae/packages/graphql-transformer-core/src/util/sanity-check.ts). Based on the docs for [deep-diff](https://github.com/flitbit/diff) I would guess that a diff was coming back as undefined and causing the error you saw.

**Comment by edwardfoyle** [comment]
The fix for this has been released in CLI v 4.32.0
